pkg/endpoint/endpoint.go

```diff
@@ -294,6 +294,14 @@ type Endpoint struct {
 	// reference to all policy related BPF
 	PolicyMap *policymap.PolicyMap `json:"-"`
 
+	// IngressPolicy contains all policies combined by each type of policy peer
+	// for Ingress enforcement.
+	IngressPolicy *policy.Policy
```

Comment:
I want to start moving fields into two  sub groups of the endpoint:
- `DesiredState`
- `RealizedState`

I'm working on a PR to do that for the configuration. I'll submit the initial version in a bit so you can base it off on it. We will want to add this policy to both `DesiredState` (calculated), and `RealizedState` (copy of when it was applied to the datapath).